python/instrumentation/openinference-instrumentation-llama-index/src/openinference/instrumentation/llama_index/_callback.py

```diff
@@ -314,90 +330,56 @@ def on_event_end(
                     f"event_type={event_type}, payload={payload}",
                 )
             if (
-                not is_straggler
-                and _is_streaming_response(response := payload.get(EventPayload.RESPONSE))
+                _is_streaming_response(response := payload.get(EventPayload.RESPONSE))
                 and response.response_gen is not None
             ):
                 response.response_gen = _ResponseGen(response.response_gen, event_data)
-                event_data.is_dispatched = True
+                is_dispatched = True
 
-        if is_straggler:
-            # This can happen if `on_event_end` is called after `on_trace_end`, e.g. in the case of
-            # the LLM span when streaming is true.
+        if not is_dispatched:
             _finish_tracing(event_data)
-            self._stragglers.pop(event_id)
 
-    def start_trace(self, trace_id: Optional[str] = None) -> None:
-        self._event_data.clear()
+    def start_trace(
+        self,
+        trace_id: Optional[str] = None,
+        *args: Any,
+        **kwargs: Any,
+    ) -> None:
+        """This is intentionally empty."""
```

Comment:
maybe explain why it's intentionally empty

Reply:
done